Lib/ufo2ft/outlineCompiler.py

```diff
@@ -978,6 +983,34 @@ def setupTable_CPAL(self):
         except ColorLibError as e:
             raise InvalidFontData("Failed to build CPAL table") from e
 
+    def setupTable_meta(self):
+        """
+        Make the meta table.
+
+        ***This should not be called externally.** Sublcasses
+        may override or supplement this method to handle the
+        table creation in a different way if desired.
+        """
+        if "meta" not in self.tables:
+            return
+
+        font = self.ufo
+        self.otf["meta"] = meta = newTable("meta")
+        ufo_meta = font.lib.get(OPENTYPE_META_KEY)
+        for key, value in ufo_meta.items():
+            if key in ["dlng", "slng"]:
+                assert isinstance(value, (list, tuple))
+                assert all(isinstance(string, str) for string in value)
```

Comment:
I think we want to explicitly raise TypeError instead of assertion error here; it's a relatively new key, make sense to validate properly